### DISCUSSION TITLE
Xbox compatibility?

### DISCUSSION THREAD

**Discussion by Nun-z** [colinkiama/UWP-Package-Installer#15] [Q&A] [answered]
Is this app theoretically compatible with Xbox devices?

**Comment by colinkiama** [comment, marked as answer]
Yeah

Envoyé de mon iPhone

**Reply by Nun-z** [reply]
Is it ok if I use this on retail mode for an Xbox?

**Reply by colinkiama** [reply]
Yes? Why wouldn't it be ok? 🤔

**Reply by Nun-z** [reply]
Don't know, sorry, someone told me to ask you for permission.  Anyway, it doesn't appear to work in retail mode, so it doesn't really matter.
